sdk/identity/identity/package.json

```diff
@@ -129,7 +129,6 @@
     "util": "^0.12.1",
     "sinon": "^9.0.2",
     "@types/sinon": "^9.0.4",
-    "mock-fs": "^4.10.4",
```

Comment:
Don't we need rush update --full when removing a dependency?

Reply:
I’ll do so! thank you!